rye/src/cli/test.rs

```diff
@@ -20,7 +20,7 @@ use crate::utils::{CommandOutput, QuietExit};
 #[derive(Parser, Debug)]
 pub struct Args {
     /// Perform the operation on all packages
-    #[arg(short, long)]
+    #[arg(short, long, default_value = "true")]
```

Comment:
I'm confused about this change. Does that not say that the flag is always true then?

Reply:
When you run `rye test` with no other arguments projects is empty and all is set to false, locate project will only return [`base_project`](https://github.com/astral-sh/rye/blob/810aa94f7498585603e6d5195c0dea302065b3c2/rye/src/pyproject.rs#L1378) and run pytest at the root when we actually want to run all projects individually and ignore the root if it is a workspace.

Reply:
I'll think of something else